src/styles/shared/reset.scss

```diff
@@ -125,22 +125,22 @@ button,
 select {
   text-transform: none;
 }
-[type="reset"],
-[type="submit"],
+[type='reset'],
+[type='submit'],
 button,
-html [type="button"] {
+html [type='button'] {
   -webkit-appearance: button;
 }
-[type="button"]::-moz-focus-inner,
-[type="reset"]::-moz-focus-inner,
-[type="submit"]::-moz-focus-inner,
+[type='button']::-moz-focus-inner,
+[type='reset']::-moz-focus-inner,
+[type='submit']::-moz-focus-inner,
 button::-moz-focus-inner {
   border-style: none;
   padding: 0;
 }
-[type="button"]:-moz-focusring,
-[type="reset"]:-moz-focusring,
-[type="submit"]:-moz-focusring,
+[type='button']:-moz-focusring,
```

Comment:
Let's reset these files to go back to double quotes. I made this ticket for us to lint scss files so that our build enforces consistent code styles

https://github.com/morehumaninternet/frontend/issues/128

Reply:
The double quotes automatically changed to single quotes after saving the file.

Reply:
Will changed vscode settings later on.